tsdb/engine/tsm1/engine.go

```diff
@@ -1990,12 +2001,14 @@ func (e *Engine) fullCompactionStrategy(group CompactionGroup, optimize bool) *c
 
 	if optimize {
 		s.description = "optimize"
+		s.name = "tsm.compact_group.optimize"
```

Comment:
Can this be `tsm1`? I think that's used in other places. If `tsm1` isn't logged in any other locations, disregard this comment. If it is, then we should specify that this is for tsm1.

Reply:
Changed to `tsm1`